File: backend/scripts/set-admin-user.js
#!/usr/bin/env node

const { PrismaClient } = require('@prisma/client');

const prisma = new PrismaClient();

async function setAdminUser() {
  const adminEmail = 'isky999@gmail.com';
  
  try {
    // Check if user exists
    const existingUser = await prisma.user.findUnique({
      where: { email: adminEmail }
    });

    if (!existingUser) {
      console.log(`❌ User with email ${adminEmail} not found. Please create the user first.`);
      return;
    }

    // Update user to admin
    const updatedUser = await prisma.user.update({
      where: { email: adminEmail },
      data: { isAdmin: true },
      select: {
        id: true,
        email: true,
        firstName: true,
        lastName: true,
        isAdmin: true
      }
    });

    console.log('✅ Admin user set successfully:');
    console.log(`   Email: ${updatedUser.email}`);
    console.log(`   Name: ${updatedUser.firstName} ${updatedUser.lastName}`);
    console.log(`   Admin: ${updatedUser.isAdmin}`);
    console.log(`   User ID: ${updatedUser.id}`);
    
  } catch (error) {
    console.error('❌ Error setting admin user:', error.message);
  } finally {
    await prisma.$disconnect();
  }
}

setAdminUser();

File: unified-python/set_admin_user.py
#!/usr/bin/env python3

import os
import sys
import logging
from sqlalchemy.orm import Session
from database import SessionLocal, User

logging.basicConfig(level=logging.INFO)
logger = logging.getLogger(__name__)

def set_admin_user():
    admin_email = 'isky999@gmail.com'
    
    db = SessionLocal()
    try:
        # Check if user exists
        existing_user = db.query(User).filter(User.email == admin_email).first()
        
        if not existing_user:
            logger.error(f"❌ User with email {admin_email} not found. Please create the user first.")
            return False
        
        # Update user to admin
        existing_user.is_admin = True
        db.commit()
        db.refresh(existing_user)
        
        logger.info("✅ Admin user set successfully:")
        logger.info(f"   Email: {existing_user.email}")
        logger.info(f"   Name: {existing_user.first_name} {existing_user.last_name}")
        logger.info(f"   Admin: {existing_user.is_admin}")
        logger.info(f"   User ID: {existing_user.id}")
        
        return True
        
    except Exception as e:
        logger.error(f"❌ Error setting admin user: {e}")
        db.rollback()
        return False
    finally:
        db.close()

if __name__ == "__main__":
    success = set_admin_user()
    sys.exit(0 if success else 1)
